runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/types/EncodedValueTypeInformation.java

```diff
@@ -18,7 +18,6 @@
 package org.apache.beam.runners.flink.translation.types;
 
 import org.apache.beam.sdk.coders.Coder;
-import org.apache.flink.annotation.PublicEvolving;
```

Comment:
This was removed to remove the flink-annotation dependency as suggested during the PR discussion.